examples/10-pgi/plot_inv_1_PGI_Linear_1D_joint_WithRelationships.py

```diff
@@ -235,11 +235,11 @@ def g(k):
 reg1 = regularization.WeightedLeastSquares(
     mesh, alpha_s=1.0, alpha_x=1.0, mapping=wires.m1
 )
-reg1.cell_weights = wr1
+reg1.set_weights(user_weights=wr1)
```

Comment:
I know it's just an example, but I think changing it to:
```Python
reg1.set_weights(cell_weights=wr1)
```
will make it clear that you are assigning cell weights.